### ISSUE TITLE
python interpretor  for linux 18:04

### ISSUE THREAD

**Issue by markh2016** [aaronduino/asciidots#62]
Please note *****  this will not run under python2.7 or below    
1:It will only run from python3 upwards  
2:When you install click you need to to this with pip3 install Click 
3: This will not work ,  "echo "alias asciidots='python $(pwd)/__main__.sh'" >> ~/.bash_profile

  Please change your instructions  under heading Ubuntu    to read 
  echo "alias asciidots='python3 $(pwd)/__main__.sh'" >> ~/.bashrc 

  Please also add   after performing above instruction   type source  ~/.bashrc 

   Now  your instructions will work and program will function as it should

**Comment by aaronduino** [comment]
> It will only run from python3 upwards
> When you install click you need to to this with pip3 install Click

Correct. I had to decide between saying "using python 3" and writing `python` in commands in the README vs just writing `python3`. I chose the former but I think I will change the README to follow the latter. Thanks for submitting the issue; it's hard to get feedback on writing, and I bet a lot of other people have been confused without saying anything.

> This will not work , `echo "alias asciidots='python $(pwd)/__main__.sh'" >> ~/.bash_profile`

Nice catch! I'll change `__main__.sh` to `__main__.py`

> Please change your instructions under heading Ubuntu to read... `~/.bashrc`

Ah, good idea.
